### ISSUE TITLE
Player camera is rotated in a random direction when snapped onto a light bridge

### ISSUE THREAD

**Issue by Luke18033** [ChaosInitiative/Portal-2-Community-Edition#179]
When a light bridge is activated where the player is standing, they will be pushed up on top of the bridge. In the original game, this is already partially broken, resetting the view pitch to face straight forward. P2CE has broken it even more, rotating the player's camera in a seeminlgly random direction. Ideally, the camera shouldn't rotate at all.

**Comment by Luke18033** [comment]
Seems to be partially fixed, though it still resets the pitch.

**Comment by CitadelCore** [comment]
Yep - the random direction thing was the same issue that caused cubes and turrets to break. This should now just be the original bug from P2.

**Comment by Luke18033** [comment]
Still an issue

**Comment by vrad-exe** [comment]
Note, iirc this issue only happens when the light bridge the player is snapped onto is the first segment from the emitter, if it's redirected through a portal then it works correctly

**Comment by ozxybox** [comment]
Fixed by @TeamSpen210 in the next update
